components/Article/EditLink/index.stories.tsx

```diff
@@ -12,4 +14,28 @@ export const Edit: Story = {
 
 export const Empty: Story = {};
 
+export const WithTranslations: Story = {
+  args: {
+    relativePath: 'get-involved/contribute.md',
+  },
+  decorators: [
+    Story => {
+      const i18nDataTranslateMode = {
+        currentLocale: {
+          code: 'xx',
+        },
+        localeMessages: {
+          'components.article.editLink.title.translate':
+            'Interested to help with translations?',
+        },
+      } as unknown as AppProps['i18nData'];
+      return (
+        <LocaleProvider i18nData={i18nDataTranslateMode}>
```

Comment:
As I mentioned before we shouldn't need to pass localeProvider here. Feel free to delete this.

Storybook is already wrapped on LocaleProvider that contains these exact translation keys.

Reply:
I was migrating the second test from here:
https://github.com/nodejs/nodejs.org/pull/5340/files#diff-d769fad9f33af5e222b1b2b7c5f92f5351ef4bba7036bef93607c593ea5ce1f9

Reply:
The original test was this:
```typescript
const i18nDataTranslateMode = {
  currentLocale: {
    code: 'xx',
  },
  localeMessages: {
    'components.article.editLink.title.translate':
      'Interested to help with translations?',
  },
} as unknown as AppProps['i18nData'];

describe('EditLink component', () => {
  it('translate mode renders correctly', () => {
    const { container } = render(
      <LocaleProvider i18nData={i18nDataTranslateMode}>
        <EditLink relativePath={relativePath} />
      </LocaleProvider>
    );
    expect(container).toMatchSnapshot();
  });
```

Reply:
Yeah, but we don't need this, to be honest.

Reply:
Removed.